api/candlepin-api-spec.yaml

```diff
@@ -3809,6 +3809,22 @@ paths:
             type: string
             format: date-time
           description: Filter jobs to those on or before this date
+        - in: query
+          name: page
```

Comment:
Can we add the `description` and `example` attributes for each paging argument here (same as [here](https://github.com/candlepin/candlepin/blob/e003c4dab79a3dc5d3623f3b72ff8af43a688a8d/api/candlepin-api-spec.yaml#L5615-L5638) I suppose)? It helps with the swagger documentation for users.